cloudinary.provisioning.account.users(null, null, "john")
  .then((response) => { console.log(response); })
  .catch((err) => { console.log(err); });